### ISSUE TITLE
[24H2] Win10 Battery flyout doesn't want to appear

### ISSUE THREAD

**Issue by CberYellowstone** [valinet/ExplorerPatcher#3640]
Windows information：24H2，26100.1591
ep version：22621.3880.66.5
when i enable the new ep taskbar10，the taskbar Win10 Battery flyout doesn't appearing，can only use Win7 Style for instead

i see that #3398 mention a similar one，but it was closed now，is that a Unfinished Feature？

**Comment by pyrates999** [comment]
Perhaps they fixed the crash, but the flyout doesn't appear with the fix.  Please change it to windows 7 fly out for battery for now.

**Comment by CberYellowstone** [comment]
> Perhaps they fixed the crash, but the flyout doesn't appear with the fix. Please change it to windows 7 fly out for battery for now.

But I had found Changelog for 22621.3007.63 says `Taskbar10: The Network and Battery flyouts on later 22621 builds onwards and Windows 10 now open instantly without issues. (97fd483)`[here](https://github.com/valinet/ExplorerPatcher/blob/5094108275056053a855190d13f250b0f942fd90/CHANGELOG.md#:~:text=Taskbar10%3A%20The%20Network%20and%20Battery%20flyouts%20on%20later%2022621%20builds%20onwards%20and%20Windows%2010%20now%20open%20instantly%20without%20issues.%20(97fd483))

hmm...

**Comment by pyrates999** [comment]
Ok, please include what settings you changed in explorer patcher properties application.  These should be screen shots.  If you can, please change it to english first.

Please include any other software you have installed that modifies the UI of windows.

**Comment by CberYellowstone** [comment]
> Ok, please include what settings you changed in explorer patcher properties application. These should be screen shots. If you can, please change it to english first.
> 
> Please include any other software you have installed that modifies the UI of windows.

![image](https://github.com/user-attachments/assets/7ae34a02-a214-4476-b79e-63b36f206e56)
![image](https://github.com/user-attachments/assets/b7b05185-1022-45d4-91ba-2b12ea73b581)
![image](https://github.com/user-attachments/assets/d4b3b37f-f6e0-4b84-ae69-c7361a9d0755)
![image](https://github.com/user-attachments/assets/29432350-ee67-48f8-8486-f5152f8b4023)
![image](https://github.com/user-attachments/assets/6e2cc585-5cef-4649-8b93-c3dd0dddcadf)
![image](https://github.com/user-attachments/assets/0640f4a8-ffe8-4e66-acac-9258398f6555)
![image](https://github.com/user-attachments/assets/1aa0b0e8-58d5-44a3-a4f2-e09d39fd3536)

**Comment by pyrates999** [comment]
You enabled the option register as shell extension, disable it by opening up the EP properties application, go to **File Explorer**, and untick **Register as shell extension**.

Here's why: https://github.com/valinet/ExplorerPatcher/wiki/Using-ExplorerPatcher-as-shell-extension

**Comment by CberYellowstone** [comment]
> You enabled the option register as shell extension, disable it by opening up the EP properties application, go to **File Explorer**, and untick **Register as shell extension**.
> 
> Here's why: https://github.com/valinet/ExplorerPatcher/wiki/Using-ExplorerPatcher-as-shell-extension

now i disabled this, but the problem still there
![动画](https://github.com/user-attachments/assets/89054516-b578-4128-88db-30155db4fafd)

**Comment by Amrsatrio** [comment]
@pyrates999 No need to ask for further info and etc. It is broken and I still haven't found a way to fix it without modifying system files.

OP, you can keep this issue open.

**Comment by randomidiot2887** [comment]
This also does not work for mine. WIndows 11 pro 24H2 Biuld 26100.ge_release.240331-1435 in 24H2. i upgraded from windows 11 pro 23H2 Biuld 226314249. how do i fix it by modifying system files?

**Comment by pyrates999** [comment]
> This also does not work for mine. WIndows 11 pro 24H2 Biuld 26100.ge_release.240331-1435 in 24H2. i upgraded from windows 11 pro 23H2 Biuld 226314249. how do i fix it by modifying system files?

Wait for a fix.  I don't think modifying system files manually is a proper fix for this.  Especially if you don't know what you're doing.

**Comment by randomidiot2887** [comment]
> > This also does not work for mine. WIndows 11 pro 24H2 Biuld 26100.ge_release.240331-1435 in 24H2. i upgraded from windows 11 pro 23H2 Biuld 226314249. how do i fix it by modifying system files?
> 
> Wait for a fix. I don't think modifying system files manually is a proper fix for this. Especially if you don't know what you're doing.

I know. But I’m just curious.

**Comment by HoodyEz** [comment]
+1

**Comment by SkyliaMontage** [comment]
I have this issue too 

Windows 11 Pro for Workstations 24H2

**Comment by Moopere** [comment]
I see there are several reports of this problem - most are being redirected back to here so I'll add my 2c:

As others have reported, using the EP taskbar and clicking on the battery in the systray notification area when its set for Win10 style produces nothing: Win11/LTSC/26100.  The win7 version works fine and its probably good enough, however this subject is mentioned in the EP wiki:

https://github.com/valinet/ExplorerPatcher/wiki/All-features#battery

Where the suggestion is to _not_ use the Win7 version but instead to use a 3rd party battery app: https://github.com/tarcode-apps/BatteryMode

I tried 'batterymode' and it is very excellent.  Really professional piece of work I'd highly recommend.

The problem though is that both battery mode and almost all comments on the web that relate to removing the original battery app no longer work with Win11.  You cannot turn off the battery icon via Win11/Settings/Taskbar/Notification Area/Turn System Icons on or off as you used to be able to do.

The result, if you use the battery mode app, is that there are two battery icons in the notification area.

So, my comment here would be a request for one of the options in EP to not only be Win10 version, Win7 version, but also 'nothing'

**Comment by TylerKweskin** [comment]
I'm also having this issue just updated and the Win10 doesn't pop out finally got the win7 bar to load after resetting multiple times but had major issue getting it to work

**Comment by Amrsatrio** [comment]
Hi all, developed a fix for this issue, coming in 69.7 (or rather 70.0?). Really sorry for the long wait.

But unfortunately there's bad news, this flyout might get removed completely in 26H1 or later.

<img width="2139" height="1126" alt="Image" src="https://github.com/user-attachments/assets/4435a651-ce19-4946-aaed-76ddf9a2c915" />

**Comment by Gismo6303** [comment]
Hello, it don't work for me too on win 11 pro 25H2

**Comment by Moopere** [comment]
> But unfortunately there's bad news, this flyout might get removed completely in 26H1 or later.

Mmm.  Thats unfortunate - but follows the general trend of MS lately.  Still, if it gets removed completely this might actually be for the best, we could then use: https://github.com/tarcode-apps/BatteryMode ("BatteryMode") a tiny app that works really well ... the current problem being that we can't turn off the currently in-built battery monitor so we end up with two.

**Comment by Shapes0** [comment]
I figured this out
Download this from Amrsatrio's repo
https://github.com/Amrsatrio/ExplorerPatcher/actions/runs/20001776900/artifacts/4789208196
This latest build has the issue patched and it's working for me

**Comment by pyrates999** [comment]
that link is not to this github repo, please don't click on it.

**Comment by Shapes0** [comment]
> that link is not to this github repo, please don't click on it.

As far as I can tell he's the one who's pushed the most recent commits to the project though? I looked through the commits and nothing looked weird to me.

**Comment by pyrates999** [comment]
I would wait until a pre-release or release is done as they need to be tested.

**Comment by Amrsatrio** [comment]
> As far as I can tell he's the one who's pushed the most recent commits to the project though?

Correct. Have been daily-ing this on my 24H2 ARM laptop and no problems aside from bugged opening animation. Perhaps I should make a new pre-release soon...

**Comment by onepointfive-REAL** [comment]
Its happening to me
